.github/workflows/prod-build.yml

```
@@ -176,9 +176,12 @@ jobs:
           REACT_APP_ENABLE_PLUS_EU: true
 
           # Document survey (https://github.com/mdn/yari/pull/6371).
-          REACT_APP_SURVEY_START: 0 # (new Date("2022-06-..")).getTime()
-          REACT_APP_SURVEY_END: 0 # (new Date("2022-06-..")).getTime()
-          REACT_APP_SURVEY_RATE: 0.05 # 5 Percent.
+          REACT_APP_SURVEY_START_CSS_CASCADE_2022: 1655208000000 # (new Date("2022-06-14 12:00:00Z")).getTime()
```

```suggestion
          REACT_APP_SURVEY_START_CSS_CASCADE_2022: 1655208000000 # (new Date("2022-06-14 12:00:00Z").getTime()
```

The timestamp is all good though :)

---

Those parenthesis come from Claas and I think they where intentional like that. Doesn't make a difference :)